community/examples/fluent-tutorial.yaml

```diff
@@ -112,7 +112,7 @@ deployment_groups:
     kind: terraform
     id: homefs
     settings:
-      network_name: "default"
+      network_id: "default"
```

Comment:
While this change is correct, I am nit picking on the fact that this blueprint does not use a pre-existing-vpc and instead is relying on the arguably incorrect vm instance default, so it is passing the network as a string to all modules that need it. 

Would this not be an opportunity to add a pre-existing-vpc at the top of the deployment group and then just use the network everywhere?

Reply:
Agreed. I will open a separate PR after merging.